src/ConnectedMode/QualityProfiles/QualityProfileDownloader.cs

```diff
@@ -52,18 +54,18 @@ internal class QualityProfileDownloader : IQualityProfileDownloader
 
         private readonly IEnumerable<Language> languagesToBind;
 
+        [ImportingConstructor]
         public QualityProfileDownloader(
             ISonarQubeService sonarQubeService,
             IBindingConfigProvider bindingConfigProvider,
             IConfigurationPersister configurationPersister,
-            ISolutionBindingOperation solutionBindingOperation,
             ILogger logger) :
             this(
                 sonarQubeService,
                 bindingConfigProvider,
                 configurationPersister,
-                solutionBindingOperation,
                 logger,
+                new SolutionBindingOperation(),
```

Comment:
The `SolutionBindingOperation` is now very simple and doesn't have any dependencies itself, so the fact that it is being used here is an implementation detail that can be hidden
-> removed from the `public` constructor and added to the `internal` constructor for testing.